docs/root/intro/arch_overview/observability/statistics.rst

```diff
@@ -42,11 +42,7 @@ become histograms as the only difference between the two representations was the
 Lazy Initialization of Stats
 ============================
 
-When :ref:`enable_lazyinit_stats <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.enable_lazyinit_stats>`
-is enabled in Bootstrap, for stats that are created using the LazyCompatibleInterface, the actual stats struct creation
-is deferred to first access of any member of that stats. E.g. when enabled, a ClusterTrafficStats structure is only
-created when any member of ClusterInfoImpl::trafficStats() is accessed.
-
-Current LazyCompatibleStats stats:
-
-* ClusterInfoImpl::trafficStats()
+When :ref:`enable_deferred_creation_stats <envoy_v3_api_field_config.bootstrap.v3.Bootstrap.enable_deferred_creation_stats>`
+is enabled in Bootstrap, for stats that are deferred creation compatible, the actual stats struct creation
+is deferred to first access of any member of that stats, i.e., instantiation only happens when an invocation on
```

Comment:
Again, please don't refer to c++ code in the docs, including structs, operators, etc. The docs audience doesn't necessarily know c++. I'd just say they are created when they are first used.